packages/app/src/cli/models/extensions/extension-instance.ts

```diff
@@ -198,6 +198,11 @@ export class ExtensionInstance<TConfiguration extends BaseConfigType = BaseConfi
     return config.build.command
   }
 
+  get watchPaths() {
+    const config = this.configuration as unknown as FunctionConfigType
+    return config.build.watch ? [config.build.watch].flat() : []
```

Comment:
should we add the defaults for javascript/rust here? I think it makes more sense

Reply:
✅